Source/items.cpp

```diff
@@ -2021,49 +2021,44 @@ int RndTypeItems(int itype, int imid)
 }
 // 421CB7: using guessed type int var_80C[512];
 
-int CheckUnique(int i, int lvl, int uper, BOOLEAN recreate)
+int CheckUnique(int i, int lvl, int uper, BOOL recreate)
 {
-	int numu;      // ebx
-	int j;         // esi
-	int idata;     // eax
-	char uok[128]; // [esp+8h] [ebp-84h]
+	int j, idata, numu;
+	BOOLEAN uok[128];
+
+	int v8;

```

Comment:
What's v8 used for? Can we rename or remove it?

Reply:
A good sign that I should have gone to bed earlier :)